Frameworks/Foundation/NSException.mm

```diff
@@ -216,7 +244,18 @@ - (HRESULT)_hresult {
         return [hresultValue unsignedIntValue];
     }
 
-    TraceWarning(L"NSException", L"Called -hresult on an NSException without an HRESULT!");
-    return E_UNEXPECTED;
+    return [[self class] _HRESULTForExceptionName:self.name];
+}
+
+- (void)_processException {
+    HRESULT hr = [self _hresult];
```

Comment:
I believe Dustin answered this question.

---
In reply to: [93302629](https://github.com/Microsoft/WinObjC/pull/1570#discussion_r93302629) [](ancestors = 93302629,93301074,93125224)